ergo-lib/src/chain/ergo_box.rs

```diff
@@ -214,6 +216,14 @@ impl IrErgoBox for ErgoBox {
     fn bytes(&self) -> Result<Vec<i8>, SigmaSerializationError> {
         Ok(self.sigma_serialize_bytes()?.as_vec_i8())
     }
+
+    fn bytes_without_ref(&self) -> Result<Vec<i8>, SigmaSerializationError> {
```

Comment:
The implementation at the time of the commit linked in the issue used `ErgoBoxCandidate::bytes` with a `TODO`: 

```
// TODO v5.0: re-implement extracting directly from `ErgoBox.bytes` array
```

Figured I'd give this solution a shot, not sure if it's right.